### PR TITLE
added issue, pr templates, welcome bot

### PR DESCRIPTION
**Pull request by Sloth-Panda**
corresponds to issue-> #23 

@RAJAGOPALAN-GANGADHARAN

### CODE REVIEW

File: .github/config.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  
+  Thank you and congrats🎉 for opening your first pull request. 🙌🙌 We will get back to you as soon as we can 😄.
+  
+  
+# Configuration for first-pr-merge - https://github.com/behaviorbot/first-pr-merge
+# Comment to be posted to on pull requests merged by a first time user
+
+firstPRMergeComment: >
+  Congrats on merging your first pull request! 🎉 All the best for your amazing open source journey ahead 🚀.
+
```
Comment (RAJAGOPALAN-GANGADHARAN) [inline .github/config.yml:30]: Please add a new line here, helps git perform its magic.

Comment (Sloth-Panda) [reply .github/config.yml:30]: @RAJAGOPALAN-GANGADHARAN  you want to go them one below the other?

Comment (Sloth-Panda) [reply .github/config.yml:30]: have resolved it

**Comment by RAJAGOPALAN-GANGADHARAN** [comment]
Great job looking for more contributions from you.

**Pull request merged**